.github/workflows/vale.yml

```
@@ -3,8 +3,7 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/*.adoc'
-      - '.github/workflows/vale.yml'
```

Like @maximiliankolb I think it's good to run a workflow if you make changes to it. I think it should be:
```suggestion
      - 'guides/common/**.adoc'
      - '.github/workflows/vale.yml'
```

---

Will give this a try.